docs/docs/get-started/basic-concepts.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Basic Concepts
+sidebar_label: Basic Concepts
+slug: '/basic-concepts'
+sidebar_position: 4
+---
+
+Package
+-----------
+
+A package defines the set up logic for a containerized backend. Packages can be accessed via Github locators.
+
+```bash 
+kurtosis run github.com/kurtosis-tech/basic-service-package
+```
+
+Packages can accept a set of parameters, defined by the package author, which enable the package consumer to modify their deployed backend at a high-level without needing to know how to configure each individual service.
+
+```bash
+kurtosis run github.com/kurtosis-tech/basic-service-package \
+  '{"service_a_count": 2, 
+    "service_b_count": 2, 
+    "service_c_count": 1,
+    "party_mode": true}'
+```
+
+Enclave
```

Comment:
I would add a sentence here to take readers to the "Advanced Concepts" section for Enclaves if they want to learn more. I also would consider adding 1 sentence about what an enclave *technically* is: a bridge network.

Reply:
I want to opt for extreme brevity here - so will add in extra links if we hit a spot where brevity comes an issue. with regards to what enclave technically is, i'd consider that advanced knowledge. for the package consumer use case they don't have to know it (the enclave also contains more than a bridge network..on docker its bridge network + files artifacts, on k8s its namespace + files artifacts...starts to get complicated understanding it once those details are presented and probably best left for advanced concepts)